testing/environments/elasticsearch/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2.3'
+services:
+  elasticsearch:
+    image: "${ELASTICSEARCH_IMAGE_REF}"
+    healthcheck:
+      test: ["CMD", "curl", "-u", "admin:testing", "-f", "https://localhost:9200", "--insecure"]
```

Comment:
how about we replace `--insecure` with `--cacert ./pki/ca/ca.crt` ?

Reply:
It fails, because the server certificate doesn't match the hostname `localhost`

Reply:
(This is also an issue with local integration testing; the only way I know other than skipping cert verification is to run the shipper itself in a container. Suggestions welcome if you know an alternative, cause I'd prefer something better.)

Reply:
We can just add a SAN to the cert to fix.  But lets leave this as `--insecure` for now and change that when we do the cert generation stuff.